01_introduction_to_algorithms/csharp/01_binary_search/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
They aren't necessary per se. Most people, though, who do cross-plat C# will find them useful. It will avoid some set up required.